app/src/main/java/ch/epfl/skysync/dataModels/flightModels/Basket.kt

```diff
@@ -0,0 +1,7 @@
+package ch.epfl.skysync.dataModels.flightModels
+
+data class Basket(val name:String, val hasDoor: Boolean)
```

Comment:
This should have an id field (even if never used by the front-end, it is needed for the database, especially in a case like ```flight.setBasket(basket)```)